modules/standard/Math.chpl

```diff
@@ -46,6 +46,37 @@ module Math {
   private use CTypes;
   private use AutoMath;
 
+  //////////////////////////////////////////////////////////////////////////
+  // Constants (included in chpldocs)
+  //
```

Comment:
Because of #5946 we can't do anything nice with sections yet.  So I've added a line to the end of the general module description, right before the first symbol:

> <img width="691" alt="Screen Shot 2023-03-31 at 3 08 51 PM" src="https://user-images.githubusercontent.com/2454710/229241677-026b3866-0fdd-4e6f-bc71-dec4c093ee43.png">

It does have a nice transition to it.  The main drawback is that the transition out of the constants isn't as obvious:

> <img width="670" alt="Screen Shot 2023-03-31 at 3 09 01 PM" src="https://user-images.githubusercontent.com/2454710/229241681-4d139a3a-09ca-478f-bff5-ceeae9b47bda.png">

I could do something more involved, like the categorization stuff we have for the [FileSystem module](https://chapel-lang.org/docs/modules/standard/FileSystem.html) today.  I think that'd probably be handy to do for the module anyways.

Reply:
Yeah, I agree that'd be nice and the current transition is not great. I don't have a preference whether that should happen in this PR or not. It'd be nice to make a note if postponing and find an owner if you are not.

Reply:
I have another few symbols I'm going to move, so I think I'll wait until the dust is a little more settled, but I've left a note on the general Math module issue list and will definitely get to it this release cycle